cloudinit/config/schema.py

```diff
@@ -169,7 +170,9 @@ class SchemaType(Enum):
         "latest": USERDATA_SCHEMA_FILE,
     },
     SchemaType.NETWORK_CONFIG: {
-        "latest": NETWORK_CONFIG_V1_SCHEMA_FILE,
+        "latest": NETWORK_CONFIG_V2_SCHEMA_FILE,
+        "1": NETWORK_CONFIG_V1_SCHEMA_FILE,
+        "2": NETWORK_CONFIG_V2_SCHEMA_FILE,
```

Comment:
The way we've named things is confusing, but this isn't really what we want here. The versioning here is for versions of the schemas themselves (which as of now and possibly forever, they aren't explicitly versioned), not the network configuration version. 

Part of me is tempted to remove this section entirely, but if we keep it we should have a `NETWORK_CONFIG_V1` and `NETWORK_CONFIG_V2` under the `SchemaType` enum with separate `latest` versions here.